.github/workflows/build.yml

```diff
@@ -20,14 +20,15 @@ on:
 
 jobs:
   universal:
-    name: Build on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    name: Build on node ${{ matrix.node_version }} and ${{ matrix.os }} with ${{ matrix.command }}
     if: ${{ github.event_name == 'push' || github.event.pull_request.draft == false }}
     runs-on: ${{ matrix.os }}
     timeout-minutes: 30
     strategy:
       matrix:
         node_version: ['12', '14']
         os: [ubuntu-latest, macOS-latest]
+        command: [build, dev-build]
```

Comment:
This added too many jobs, shall we just add a separate workflow to confirm node14+macOS? And not all steps in build.yml is needed, too :)